save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/WelcomeUtils.kt

```diff
@@ -24,11 +24,13 @@ import web.cssom.*
 /**
  * @param oauthProviders
  * @param primaryColor color of a shield
+ * @param continueLink link for `continue` button
  */
 @Suppress("TOO_LONG_FUNCTION")
 internal fun ChildrenBuilder.inputCredentialsView(
     oauthProviders: List<OauthProviderInfo>,
     primaryColor: Colors,
+    continueLink: String,
```

Comment:
why do you need an argument if it is always hardcoded?

Reply:
Is it always hardcoded?

I guess user unauthorised user could visit both `/vuln/list` and `/projects`

Reply:
<img width="954" alt="screenshot" src="https://github.com/saveourtool/save-cloud/assets/35039155/d1ff0c1e-cfeb-470b-bb2b-4f3403186c3b">